src/main/dynamo/pagedIterator.ts

```diff
@@ -51,29 +56,32 @@ export async function* pagedIterator<T, U extends TaggedModel>(
   jayz?: JayZ
 ): AsyncGenerator<PageResults<U>, PageResults<U>> {
   let pendingOperation: T | undefined = buildOperation(options)
+  let cursor: Cursor | undefined
 
   while (pendingOperation !== undefined) {
-    const response: DynamoDB.DocumentClient.QueryOutput = await executeOperation(
-      pendingOperation
-    )
+    try {
+      const response: DynamoDB.DocumentClient.QueryOutput = await executeOperation(
+        pendingOperation
+      )
 
-    if (response.LastEvaluatedKey !== undefined) {
-      pendingOperation = buildOperation({
-        ...options,
-        cursor: response.LastEvaluatedKey
-      })
-    } else {
-      pendingOperation = undefined
-    }
+      if (response.LastEvaluatedKey !== undefined) {
+        cursor = response.LastEvaluatedKey
+        pendingOperation = buildOperation({ ...options, cursor })
+      } else {
+        pendingOperation = undefined
+      }
 
-    const maybeDecryptedItems = await decryptOrPassThroughItems(
-      jayz,
-      response.Items || []
-    )
+      const maybeDecryptedItems = await decryptOrPassThroughItems(
```

Comment:
If this method instead processed a single item, it'd allow us to walk the items array here. That'd be nicer because if we failed to decrypt a record, we could still successfully process the rest of the records contained within this page. 

As it is now, if we hit a record we can't decrypt (e.g. it might've been encrypted with a bad key), we'll give up on all the records contained within this page.